### DISCUSSION TITLE
Date and timestamp meta data of exported segments

### DISCUSSION THREAD

**Discussion by lindig** [mifi/lossless-cut#770] [Q&A]
I am capturing videos with an iPhone (*.MOV) and cut out segments. Is it possible that the segments reflect the date/time in their meta data as if they were taken as individual recordings? For example:

* Original video taken on 2021-06-17 13:00 to 13:03 (3min)
* First segment 0:00 to 1:00 should have start date 2021-06-17 13:00
* Second segment 2:00-3:00 should have start date 2021-06-17 13:02

This way I would preserve the date and time individual segments were taken. If this is already the case, please tell me, otherwise: is this a sensible feature?

**Comment by mifi** [comment]
I have just implemented this. See #468

**Reply by lindig** [reply]
Great. I’ll take a look

**Reply by Heru-MS** [reply]
> 
> 
> I have just implemented this. See #468

Hello mifi,
I just got the latest version and tested it by cutting 4 segments out of a 4 GB-File. All 4 show the same creation date/time. Is there another tag for this, or must this feature be activated somewhere?

I would like to have the option to name the different segments according to their creation date/time. This would be a great help when mixing clips from different cameras...

**Reply by mifi** [reply]
LosslessCut will only update the "file modified date" for the output files. can you check if modified date gets offset correctly? It is not possible to update the "file created date" unfortunately, and any MP4 metadata tags will not be updated.

**Comment by lulalala** [comment]
@mifi Thanks for the feature. It works well for videos.
However I found that for Capture frame (i.e. save as JPEG), its updated time semantic does not align with the segment exports.

For example, I'm working on 092.mp4, and I choose the "addition" with "end" as the offset strategy:

![B2025-05-01 14_15_11-2025 RubyKaigi](https://github.com/user-attachments/assets/64a83297-b6e1-4fd2-9fae-1ed51eb0bafd)

The video segments are correctly offset to be less than the source file. However the screenshot offsets are going the other direction.

**Reply by mifi** [reply]
I've tried to improve this. you can try it in the next nightly build
